nemoguardrails/library/autoalign/actions.py

```diff
@@ -198,7 +201,10 @@ async def autoalign_factcheck_infer(
             async for line in response.content:
                 resp = json.loads(line)
                 if resp["task"] == "factcheck":
-                    return float(resp["response"][17:])
+                    try:
+                        return float(resp["response"][17:])
+                    except ValueError as _:
+                        return 1.0
```

Comment:
did a path cause a reason for this ValueError check? seems bit unsafe to return 1, since we always expect this parse to work

Reply:
For cases where no documents are given, the resp is like "No documents provided" so it is to handle those cases

Reply:
I see, make sense. Could we instead do a startswith("Factcheck Score: ") check? to make it more explicit